client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx

```diff
@@ -248,17 +357,21 @@ const SiteSetupList = ( {
 							{ currentTask.description }
 						</p>
 						<div className="site-setup-list__task-actions task__actions">
-							<Button
-								className="site-setup-list__task-action task__action"
-								primary
-								onClick={ () => startTask( dispatch, currentTask, siteId ) }
-								disabled={
-									currentTask.isDisabled ||
-									( currentTask.isCompleted && currentTask.actionDisableOnComplete )
-								}
-							>
-								{ currentTask.actionText }
-							</Button>
+							{ currentTask.actionText && (
+								<Button
+									className="site-setup-list__task-action task__action"
+									primary
+									onClick={ () =>
+										startTask( dispatch, currentTask, siteId, advanceToNextIncompleteTask )
```

Comment:
Non-blocking: If this is getting too long, we can change the startTask signature to an options object.

Reply:
Noted.